thunder/core/transforms.py

```diff
@@ -3073,4 +3076,100 @@ def backward_fn(saved_for_backward, cotangents):
     _update_backward_with_new_saved_for_backward(backward_trace, only_used_bw_saved_for_backward)
     forward_trace.set_provenance(TraceProvenance("Augmented forward pass"))
     backward_trace.set_provenance(TraceProvenance("Backward pass"))
+
+    enable_saved_for_backward_recomputation: None | bool = get_compile_option(
```

Comment:
QQ: Could you tell me why this allows `None`?

Reply:
When the flag is not used this results in a `None` you're right tho that it could be initialized with a bool default value, say `False`